src/pages/article-view-page.js

```diff
@@ -44,26 +67,162 @@ class ArticleView extends LitElement{
                 margin: 0 auto;
                 
             }
+            .belowComment-section{
+                padding:10px;
+                display: flex;
+            }
             .article-body-container{
                 border-bottom:1px solid grey;
                 width:80%;
                 margin:0 auto;
                 padding: 10px 0 50px 0;
             }
             .comment-section{
-                width:80%;
-                margin:0 auto;
+                width:70%;
+                margin: 10px auto;
+                border-radius: 5px;
+                background-color:#F5F5F5;
+                border:1px solid  #888888bb;
+
             }
-            .right{
-                float:right;
+            .cmt-author{
+                vertical-align:middle;
+                padding-top:8px;
+                padding-left:6px;
+                font-size:12px;
             }
+            .avatar {
+                vertical-align: middle;
+                width: 40px;
+                height: 40px;
+                border-radius: 50%;
+              }
+              .btn-wrapper{
+                flex-grow: 1;
+                display :flex;
+                justify-content: flex-end;
+              }
+          
         `;
     }
 
     static get properties(){
         return {
-            dataLoaded: {type:Boolean}
+            dataLoaded: {type:Boolean},
+            displayComments: {type:Array},
+            userImage:{type:String}
+        }
+    }
+    handleChange(e){
+       this[e.target.name] = e.target.value;
+    }
+
+    fetchComment(){
+        fetch(`https://conduit.productionready.io/api/articles/${this.slug}/comments`, {
+            method: "GET",
+            headers: {
+              "Content-Type": "application/json",
+              "Accept": "appication/json",
+              
+            },
+        
+            
+          })
+          .then(response => {
+            if (!response.ok) throw response;
```

Comment:
What is the significance of this statement?

Reply:
to determine if we receive error from remote server, response,ok return false and hence throwing it to page not found